addons/mail/static/src/new/messaging.js

```diff
@@ -307,10 +310,15 @@ export class Messaging {
         } else {
             thread.messages.push(id);
         }
+        this.sortThreadMessages(thread);
```

Comment:
it feels like this is wrong for chatter messages: the code above sort then in reverse order.

Also, note that this is a quadratic behaviour: adding n messages means that we have to sort the messages list n times. probably not a big deal in practice, but usually we can do better

Reply:
ok, it seems like it works thanks to the order props. not a fan of having to reverse it each time, but it works

Reply:
Agree, we could find a way to save both orders.
Note that chatter can be in a chat window, and the order should be the same as with channels. Hence why we can't just keep a single ordering of messages for a given thread.